x/bank/types/msgs.go

```diff
@@ -69,83 +62,6 @@ func (msg MsgSend) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{from}
 }
 
-// Rosetta interface
-func (msg *MsgSend) ToOperations(withStatus bool, hasError bool) []*types.Operation {
```

Comment:
if one wanted to create a rosetta operation from a message do I need to implement a specific interface? how does that work now?

Reply:
no no, this implementation supports everything out of the box. all messages can be sent.